docs/api-reference/components/request-context.md

```diff
@@ -149,33 +145,6 @@ css`
 </>
 ```
 
-## `useId`
```

Comment:
This doc is correct. Looks like a sync problem. We need to keep this documentation